security/webauthn/src/test/java/MySqlWebAuthnIT.java

```diff
@@ -0,0 +1,32 @@
+import io.quarkus.test.bootstrap.MySqlService;
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.Container;
+import io.quarkus.test.services.QuarkusApplication;
+
+@QuarkusScenario
+public class MySqlWebAuthnIT extends AbstractWebAuthnTest {
+
+    private static final String MYSQL_USER = "quarkus_test";
+    private static final String MYSQL_PASSWORD = "quarkus_test";
+    private static final String MYSQL_DATABASE = "quarkus_test";
+    private static final int MYSQL_PORT = 3306;
+
+    @Container(image = "${mysql.80.image}", port = MYSQL_PORT, expectedLog = "Only MySQL server logs after this point")
```

Comment:
Optionally:
we don't have these things very much defined, but when we use something, it's harder to stop using it later, because you don't know why this specific image was used. So I'd like to propose to use `mysql.upstream.80.image` instead, because it's easier to switch from upstream to product than otherwise when there is such requirement.

Reply:
done

Reply:
I've come back to the origin image because works, with MySQL upstream image there some SQL errors that I had not time to investigate.